src/NServiceBus.AcceptanceTests/DelayedDelivery/When_deferring_a_message_to_the_past.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace NServiceBus.AcceptanceTests.DelayedDelivery
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Features;
+    using NUnit.Framework;
+
+    public class When_deferring_a_message_to_the_past : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_deliver_message_now()
```

Comment:
by testing for an exception, this would be an unit test which does not verify the complete behavior of the send API. The test verifies that messages "sent to the past" are delivered immediately. I wanted to avoid the word immediately because it can cause confusion regarding "immediate dispatch" which is a different API.